sql/core/benchmarks/IntervalBenchmark-jdk11-results.txt

```diff
@@ -1,29 +1,39 @@
+2 units w/o interval                                654            660          10          1.5         653.5       1.1X
+3 units w/ interval                                1319           1325          10          0.8        1318.6       0.6X
+3 units w/o interval                               1330           1332           3          0.8        1329.7       0.6X
+4 units w/ interval                                1723           1725           4          0.6        1722.8       0.4X
+4 units w/o interval                               1723           1725           2          0.6        1723.1       0.4X
+5 units w/ interval                                1906           1921          15          0.5        1906.4       0.4X
+5 units w/o interval                               1902           1918          20          0.5        1901.9       0.4X
+6 units w/ interval                                2109           2118           8          0.5        2108.8       0.4X
+6 units w/o interval                               2102           2105           4          0.5        2102.3       0.4X
+7 units w/ interval                                2324           2325           1          0.4        2324.5       0.3X
+7 units w/o interval                               2381           2384           3          0.4        2380.9       0.3X
+8 units w/ interval                                2460           2483          21          0.4        2459.6       0.3X
+8 units w/o interval                               2452           2459          10          0.4        2452.3       0.3X
+9 units w/ interval                                2714           2720           9          0.4        2714.3       0.3X
+9 units w/o interval                               2707           2719          10          0.4        2707.4       0.3X
+10 units w/ interval                               2860           2863           3          0.3        2859.8       0.3X
+10 units w/o interval                              2853           2863          11          0.4        2852.9       0.3X
+11 units w/ interval                               3156           3160           7          0.3        3155.5       0.2X
+11 units w/o interval                              3169           3179          10          0.3        3168.8       0.2X
+
+OpenJDK 64-Bit Server VM 11.0.7+10-post-Ubuntu-2ubuntu218.04 on Linux 4.15.0-1063-aws
+Intel(R) Xeon(R) CPU E5-2670 v2 @ 2.50GHz
+make_interval():                          Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
+------------------------------------------------------------------------------------------------------------------------
+prepare make_interval()                            3396           3410          13          0.3        3395.6       1.0X
+make_interval(0,1,2,3,4,5,50.123456)                132            141           8          7.6         132.3      25.7X
```

Comment:
I had to remove spaces otherwise results in the table are shifted so much to the right.

Reply:
This PR https://github.com/apache/spark/pull/28906 fixes the issue.